src/modules/fancyzones/editor/FancyZonesEditor/Models/LayoutModel.cs

```diff
@@ -260,15 +288,30 @@ public static ObservableCollection<LayoutModel> LoadCustomModels()
 
                         JsonElement.ArrayEnumerator zonesEnumerator = info.GetProperty(ZonesJsonTag).EnumerateArray();
                         IList<Int32Rect> zones = new List<Int32Rect>();
+
+                        bool error = false;
                         while (zonesEnumerator.MoveNext())
                         {
                             int x = zonesEnumerator.Current.GetProperty(XJsonTag).GetInt32();
                             int y = zonesEnumerator.Current.GetProperty(YJsonTag).GetInt32();
                             int width = zonesEnumerator.Current.GetProperty(WidthJsonTag).GetInt32();
                             int height = zonesEnumerator.Current.GetProperty(HeightJsonTag).GetInt32();
+
+                            if (width <= 0 || height <= 0)

```

Comment:
`lastWorkAreaWidth` and `lastWorkAreaHeight` probably should be checked here too.

Reply:
Done